.github/workflows/interface_push_to_main.yml

```diff
@@ -0,0 +1,25 @@
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
+
+name: Interface > Push to Main
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: deploy
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+  APP_ID: app_e77e446e-754a-433d-8f8f-fe339d699a75
+
```

Comment:
We can remove because the interface does not use Docker and the app_id is hardcoded in `interface_deploy.yml`.
```suggestion
```